src/DSManager.cpp

```diff
@@ -565,7 +844,7 @@ DSManager::~DSManager()
     {
         saveSnapshots(snapshots_output_file);
     }
-    onTerminate();
+    //onTerminate();
```

Comment:
I understand that this has been commented because `onTerminate` is now called directly on `main` function. If so, please remove this commented line.